libraries/AC_InputManager/AC_InputManager_Heli.cpp

```diff
@@ -119,4 +122,34 @@ float AC_InputManager_Heli::get_pilot_desired_collective(int16_t control_in)
     return collective_out;
 }
 
+// parameter_check - check if input manager specific parameters are sensible
+bool AC_InputManager_Heli::parameter_check(char* fail_msg, uint8_t fail_msg_len) const
+{
+    // returns false if STAB_COL_1 is outside of range
+    if ((_heli_stab_col_min < 0) || (_heli_stab_col_min > 100)){
+        hal.util->snprintf(fail_msg, fail_msg_len, "IM_STAB_PCT_COL_1 out of range");
+        return false;
+    }
+
+    // returns false if STAB_COL_1 is outside of range
```

Comment:
Comment is out-of-date.  Other similar comments could also be fixed.

One of the nice things about the percentage-change is that it looks like a nice cleanup here would be to create a structure to iterate through a static list, rather than this copy-paste block.

Something like:
```
const struct StabCheck {
   const char *name;
   uint8_t value;
} stab_checks {
  {"IM_STAB_PCT_COL_1", _heli_stab_col_min },
  {"IM_STAB_PCT_COL_2", _heli_stab_col_max },
 ...
};

for (uint8_t i=0; i<ARRAY_SIZE(stab_checks); i++) {
    const StabCheck check = stab_checks[i];
    if ((check.value < 0) || (check.value > 100)){
        hal.util->snprintf(fail_msg, fail_msg_len, "%s out of range", check.name);
        return false;
    }
}
```

... give or take compiling....